pkgs/applications/networking/ps2client/default.nix

```
@@ -0,0 +1,25 @@
+{ fetchFromGitHub, stdenv }:
+
+stdenv.mkDerivation rec {
+  version = "unstable-2018-10-18";
+  name = "ps2client-${version}";
+
+  src = fetchFromGitHub {
+    owner = "ps2dev";
+    repo  = "ps2client";
+    rev = "92fcaf18aabf74daaed40bd50d428cce326a87c0";
+    sha256 = "1rlmns44pxm6dkh6d3cz9sw8v7pvi53r7r5r3kgwdzkhixjj0cdg";
+  };
+
+  installPhase = ''
+    make PREFIX=$out install
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Desktop clients to interact with ps2link and ps2netfs";
+    homepage = https://github.com/ps2dev/ps2client;
+    license = "BSD-like";
```

We have a list of licenses, you can find them here: https://github.com/NixOS/nixpkgs/blob/master/lib/licenses.nix

In that particular case, it appears to be a BSD 3-Clause:

```suggestion
    license = licenses.bsd3;
```